### DISCUSSION TITLE
How can i setup CredentialsProvider with prisma adapter?

### DISCUSSION THREAD

**Discussion by Mr-Michetti** [nextauthjs/next-auth#3364] [Help] [answered]
I don't understand how the [nextauth ...].js page should be set if I want to use CredentialsProvider and PrismaAdapter. 
Anyone have an example?
Thanks

**Comment by balazsorban44** [comment, marked as answer]
There is no built-in way, we usually don't recommend setting up credentials with your database because of the security implications most people aren't ware of. That said there are many tutorials out there.

**Reply by vallerydelexy** [reply]
where can i read more about this "security implications most people aren't ware of"?
